pkg/identityserver/application_access.go

```diff
@@ -67,7 +70,22 @@ func (is *IdentityServer) createApplicationAPIKey(ctx context.Context, req *ttnp
 	}
 	key.Key = token
 	events.Publish(evtCreateApplicationAPIKey(ctx, req.ApplicationIdentifiers, nil))
-	// TODO: Send notification email (https://github.com/TheThingsNetwork/lorawan-stack/issues/72).
+	members := map[*ttnpb.OrganizationOrUserIdentifiers]*ttnpb.Rights{}
```

Comment:
Not quite sure if we should email all members about this. We can also just send an email to the `ContactInfo`s of the entity (possibly adding an extra "SECURITY" contact type).

Reply:
Same goes for the other places where this happens